sbt-scalajs-bundler/src/main/scala/scalajsbundler/util/JS.scala

```diff
@@ -79,17 +93,35 @@ object JS {
 
   def `new`(ctor: JS, args: JS*): JS = JS(New(ctor.tree, args.map(_.tree).to[List]))
 
-  def toJson(obj: JS): String = show(obj.tree, isStat = false)
-
-  def show(tree: Tree, isStat: Boolean = true): String = {
-    val writer = new java.io.StringWriter
-    val printer = new Printers.JSTreePrinter(writer)
-    printer.printTree(tree, isStat)
-    writer.toString
-  }
-
   private val identifierSeq = new AtomicInteger(0)
   private def freshIdentifier(): String =
     s"x${identifierSeq.getAndIncrement()}"
 
 }
+
+final class JSON(tree: Tree) extends JSLike(tree)
```

Comment:
That looks perfect excepted that I would put this constructor private so that users have to use the smart constructors defined in tho `JSON` object.